backend/app/usecase/shortlink/creator_test.go

```diff
@@ -46,7 +44,7 @@ func TestShortLinkCreatorPersist_CreateShortLink(t *testing.T) {
 					ExpireAt: &now,
 				},
 			},
-			alias: &alias,
+			alias: alias,
```

Comment:
```suggestion
			alias: "220uFicCJj",
```
We can put the alias value "220uFicCJj" here direct